src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -153,4 +144,33 @@ private function getConfigForPath(array $config, string $path, string $alias)
 
         return $config;
     }
+
+    private function getConfigForExtension(ExtensionInterface $extension, ContainerBuilder $container): array
+    {
+        $extensionAlias = $extension->getAlias();
+
+        $extensionConfig = [];
+        foreach ($container->getCompilerPassConfig()->getPasses() as $pass) {
+            if ($pass instanceof ValidateEnvPlaceholdersPass) {
+                $extensionConfig = $pass->getExtensionConfig();
+                break;
+            }
+        }
+
+        if (isset($extensionConfig[$extensionAlias])) {
+            return $extensionConfig[$extensionAlias];
```

Comment:
now im unsure if `resolveValue()` was already missing here :) eg. config is given, but also inherits a default node which takes a parameter/env as default value

feel free to test :)

Reply:
you're right, this is inconsistent. I wonder if I should fix it here or create a follow-up PR